### PR TITLE
HDDS-1421. Avoid unnecessary object allocations in TracingUtil. Contr…

### PR DESCRIPTION
**Pull request by arp7**
…ibuted by Arpit Agarwal.

Change-Id: I3fd1b59447005cc62ae217c4def86a9df81944b1

### CODE REVIEW

File: hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/tracing/TracingUtil.java [hunk 1]
```diff
@@ -33,6 +33,8 @@
  * Utility class to collect all the tracing helper methods.
  */
 public final class TracingUtil {
+
```
Comment (hadoop-yetus) [inline hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/tracing/TracingUtil.java]: whitespace:end of line

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 28 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -1 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ trunk Compile Tests _ |
| +1 | mvninstall | 1023 | trunk passed |
| +1 | compile | 48 | trunk passed |
| +1 | checkstyle | 23 | trunk passed |
| +1 | mvnsite | 42 | trunk passed |
| +1 | shadedclient | 728 | branch has no errors when building and testing our client artifacts. |
| +1 | findbugs | 72 | trunk passed |
| +1 | javadoc | 45 | trunk passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 43 | the patch passed |
| +1 | compile | 35 | the patch passed |
| +1 | javac | 35 | the patch passed |
| +1 | checkstyle | 16 | the patch passed |
| +1 | mvnsite | 35 | the patch passed |
| -1 | whitespace | 0 | The patch has 1 line(s) that end in whitespace. Use git apply --whitespace=fix <<patch_file>>. Refer https://git-scm.com/docs/git-apply |
| +1 | shadedclient | 734 | patch has no errors when building and testing our client artifacts. |
| +1 | findbugs | 74 | the patch passed |
| +1 | javadoc | 37 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 63 | common in the patch passed. |
| +1 | asflicense | 29 | The patch does not generate ASF License warnings. |
| | | 3156 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/hadoop-multibranch/job/PR-722/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/722 |
| Optional Tests |  dupname  asflicense  compile  javac  javadoc  mvninstall  mvnsite  unit  shadedclient  findbugs  checkstyle  |
| uname | Linux 75a5e346cd64 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 586826f |
| maven | version: Apache Maven 3.3.9 |
| Default Java | 1.8.0_191 |
| findbugs | v3.1.0-RC1 |
| whitespace | https://builds.apache.org/job/hadoop-multibranch/job/PR-722/1/artifact/out/whitespace-eol.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-722/1/testReport/ |
| Max. process+thread count | 411 (vs. ulimit of 5500) |
| modules | C: hadoop-hdds/common U: hadoop-hdds/common |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-722/1/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by elek** [comment]
Merged to the trunk. Thanks @arp7 the contribution.

**Pull request closed without merging**